dspace-api/src/main/java/org/dspace/app/sherpa/v2/SHERPAJournal.java

```diff
@@ -21,7 +22,7 @@
  *
  * @author Kim Shepherd
  */
-public class SHERPAJournal {
+public class SHERPAJournal implements Serializable {
```

Comment:
@Raf-atmire Classes have been made implementing Serializable because of following exception arisen once we put org.dspace.app.sherpa.SHERPAService#searchByJournalISSN under cache control

ERROR 79d1d231-5046-4270-8d03-b99c02607489 c2efcf84-8a20-491e-bd20-8c4d29e04dd3 org.dspace.app.sherpa.v2.SHERPAResponse @ Failed to parse SHERPA response
org.json.JSONException: Attempted read from closed stream.
at org.json.JSONTokener.next(JSONTokener.java:201) ~[json-20180130.jar:?]
at org.json.JSONTokener.nextClean(JSONTokener.java:292) ~[json-20180130.jar:?]
at org.json.JSONObject.<init>(JSONObject.java:213) ~[json-20180130.jar:?]
at org.dspace.app.sherpa.v2.SHERPAResponse.parseJSON(SHERPAResponse.java:99) ~[dspace-api-7.3-SNAPSHOT.jar:?]
at org.dspace.app.sherpa.v2.SHERPAResponse.<init>(SHERPAResponse.java:79) ~[dspace-api-7.3-SNAPSHOT.jar:?]
at org.dspace.app.sherpa.SHERPAService.performRequest(SHERPAService.java:266) ~[dspace-api-7.3-SNAPSHOT.jar:?]
at org.dspace.app.sherpa.SHERPAService.searchByJournalISSN(SHERPAService.java:98) ~[dspace-api-7.3-SNAPSHOT.jar:?]
at org.dspace.app.sherpa.SHERPAService$$FastClassBySpringCGLIB$$3fe0dc76.invoke(<generated>) ~[dspace-api-7.3-SNAPSHOT.jar:?]
at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.3.18.jar:5.3.18]
at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:783) ~[spring-aop-5.3.18.jar:5.3.18]